app/services/usps_confirmation_maker.rb

```diff
@@ -20,6 +20,7 @@ def perform
       otp_fingerprint: Pii::Fingerprinter.fingerprint(otp),
     )
 
+    Throttler::IsThrottledElseIncrement.call(profile&.user&.id, :idv_send_letter)
```

Comment:
Should we avoid calling if `profile` or `profile.user` are `nil`? A brief look at the implementation of [`IsThrottledElseIncrement#call`](https://github.com/18F/identity-idp/blob/c34482ab0aa9647e1fa783bf6362498bc9956372/app/services/throttler/is_throttled_else_increment.rb#L4) has me thinking it may not gracefully handle `nil` passed as a `user_id`.